values-schema.yaml

```diff
@@ -1047,20 +1065,24 @@ definitions:
     description: Grant team permissions to modify certain configuration parameters.
     type: object
     properties:
-      Service:
-        title: Service
+      service:
```

Comment:
Please define mutations at `values-changes.yaml`

Reply:
not needed, those are for changing existing value repos. Non breaking additions are expected.